CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ Changelog
   * interface_channel_group (@chrisvanheuveln)
   * interface_portchannel (@saichint)
   * portchannel_global (@saichint)
+* Spanning Tree
```

Comment:
This is in the wrong section.  Please move it up under the `Unreleased` section.